spec/student.spec.js

```diff
@@ -0,0 +1,66 @@
+import Student from '../src/student.js'
+import { Organization } from '../src/organization.js'
+import Cohort from '../src/cohort.js'
+
+describe('student', () => {
+  let myOrganization
+  let newCohort
+
+  beforeAll(() => {
+    myOrganization = new Organization('Boolean')
+    myOrganization.addBranch('UK')
+    newCohort = new Cohort('My TestCohort')
+    myOrganization.branches[0].addCohort(newCohort)
+  })
+
+  it('creation works', () => {
+    const expected = {
+      studentID: myOrganization.studentCounter + 1,
+      firstName: 'Nathan',
+      lastName: 'The Wise',
+      email: 'no@email.com',
+      githubUsername: 'nwise',
+      inCohort: {}
+    }
+    const test = new Student(
+      'Nathan',
+      'The Wise',
+      'no@email.com',
+      'nwise',
+      myOrganization
+    )
+
+    expect(JSON.stringify(test)).toEqual(JSON.stringify(expected))
```

Comment:
Super neat way of testing your class information and using JSON.stringify to make it readable.